frontend/admin/src/js/components/poolCandidate/ViewPoolCandidate/ViewPoolCandidatePage.tsx

```diff
@@ -197,7 +201,12 @@ export const ViewPoolCandidate = ({
           </>
         )}
       </h2>
-      {mainContent}
+      <TableOfContents.Wrapper>
```

Comment:
I think the sidebar needs to be pulled up a bit so that the title is even with "Application's profile snapshot"
![image](https://user-images.githubusercontent.com/8978655/191545258-87f7832a-7321-49ef-8a01-c7e5394e8c95.png)

Reply:
Good catch, I have updated a couple of the styles in [adjust styles to match mockup](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3957/commits/6e52c7ce6ff7650d82fd9646201503eadedffdb5)